### PR TITLE
Update module.md and package.md

### PR DESCRIPTION
**Pull request by ayeshLK**
Part of: [#6224](https://github.com/ballerina-platform/ballerina-library/issues/6224)

## Examples
N/A

## Checklist
- [X] Linked to an issue
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Updated the spec~
- [ ] ~Checked native-image compatibility~

### CODE REVIEW

File: ballerina/Module.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+## Compatibility
+
+|                        |   Version    |
+|:-----------------------|:------------:|
+| Guidewire InsuranceNow | **2023.3 (Innsbruck)** |
+
+## Setup guide
+
+To use the Ballerina Guidewire InsuranceNow connector, access to the Guidewire API is required. Access is granted to partners or customers of the Guidewire platform. To become a partner or customer, reach out to Guidewire's sales team via the [Guidewire website](https://www.guidewire.com/contact-us/). For detailed instructions on obtaining authentication/authorization keys, please go through the [portal development guide](https://docs.guidewire.com/cloud/in/20233/portaldev/PortalDevelopment/topics/c_overview_portal_development.html).
```
Comment (ayeshLK) [inline ballerina/Module.md:11]: Currently we don't have access to a guidewire account. Hence, we could not add the screenshots for the setup guide.

File: ballerina/Module.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+[Guidewire InsuranceNow](https://www.guidewire.com/products/insurancenow) is a cloud-based insurance platform offering comprehensive tools for policy, billing, and claims management, designed to streamline operations and improve customer service in the insurance industry.
+
+The `ballerinax/guidewire.insnow` package provides a client API to connect to Guidewire InsuranceNow cloud API using Ballerina.
+
+## Compatibility
+
+|                        |   Version    |
+|:-----------------------|:------------:|
+| Guidewire InsuranceNow | **2023.3 (Innsbruck)** |
```
Comment (NipunaRanasinghe) [inline ballerina/Module.md]: IMO we dont need to have a single entry table of this. Instead we can mention the supported API version in the overview, similar to other connectors.

File: ballerina/Package.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+Create an `insnow:Client` object with your domain URL and relevant authentication options.
+
+```ballerina
+configurable string username = ?;
+configurable string password = ?;
+
+insnow:ConnectionConfig insnowConfig = {
+    auth: {
+        username: username,
+        password: password
```
Comment (NipunaRanasinghe) [inline ballerina/Package.md]:
```suggestion
        username,
        password
```

File: ballerina/Package.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        },
+        {
+            id: "tag-auto-insurance",
+            name: "Auto Insurance",
+            tagTemplateIdRef: "template-auto-ins"
+        }
+    ],
+    templateId: "template-insurance-application"
+};
+_ = check insuranceNow->/applications/["123"]/documents.post(attachment);
```
Comment (NipunaRanasinghe) [inline ballerina/Package.md]:
```suggestion
};

_ = check insuranceNow->/applications/["123"]/documents.post(attachment);
```

File: ballerina/Module.md (hunk 2 above)
Comment (ayeshLK) [reply ballerina/Module.md]: Fixed with [3ffa97ac36318fcdd70cca1fec891f1694cb734c](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/pull/12/commits/3ffa97ac36318fcdd70cca1fec891f1694cb734c)

File: ballerina/Module.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
+## Overview
+
+[Guidewire InsuranceNow](https://www.guidewire.com/products/insurancenow) is a cloud-based insurance platform offering comprehensive tools for policy, billing, and claims management, designed to streamline operations and improve customer service in the insurance industry.
+
+The `ballerinax/guidewire.insnow` package provides a client API to connect to Guidewire InsuranceNow cloud API using Ballerina.
+
+The Ballerina Guidewire InsuranceNow connector is compatible with the Guidewire InsuranceNow `2023.3 (Innsbruck)` version.
```
Comment (NipunaRanasinghe) [inline ballerina/Module.md]:
```suggestion
The Ballerina Guidewire InsuranceNow connector is compatible with the Guidewire InsuranceNow `2023.3 (Innsbruck)` REST API version.
```

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**